examples/demo/config/development.yaml

```diff
@@ -156,3 +156,35 @@ auth:
     expiration: 604800 # 7 days
 # </snip>
 
+# Scheduler Jobs Configuration
+# <snip id="configuration-scheduler">
+scheduler:
+  # Location of shipping the command stdout and stderr.
+  output: stdout
+  # A list of jobs to be scheduled.
+  jobs:
+      # The name of the job.
+    - name: "Run command"
+      # The type of job.
+      shell:
```

Comment:
I think it should be just:

```
run: "...command.."
shell: true | false
```

Reply:
Done: https://github.com/loco-rs/loco/pull/735/commits/f85601b4d1d5a5d0c84fbad6b1c77a87edc6cabc